app/models/concerns/hyrax/ability/work_ability.rb

```diff
@@ -15,9 +15,24 @@ def work_roles
             doc = permissions_doc(id)
             all_work_types_and_files.include?(doc.hydra_model)
           end
-        elsif work_depositor?
+        elsif work_depositor? || admin_set_with_deposit?
           can %i[create], all_work_types_and_files
         end
+
+        # OVERRIDE HYRAX 3.5.0 to return false if no ids are found
+        # @return [Boolean] true if the user has at least one admin set they can deposit into.
+        def admin_set_with_deposit?
```

Comment:
The `#admin_set_with_deposit?` method is defined inside the `#work_roles` method

Reply:
oh man, good catch! 🙈